crates/ruff_linter/src/rules/flake8_commas/rules/trailing_commas.rs

```diff
@@ -27,31 +27,31 @@ enum TokenType {
 
 /// Simplified token specialized for the task.
 #[derive(Copy, Clone)]
-struct Token {
+struct RuleToken {
```

Comment:
What's the motivation for renaming the type here? I'm asking because I find `RuleToken` a somewhat awkward name.

Reply:
Mainly to avoid conflicts with the actual `Token` stored on the program.

Reply:
What about `SimpleToken` ?

Reply:
I went with `SimpleToken` but feel free to suggest something else.